Examples.md

```diff
@@ -21,6 +21,20 @@ int main(void) {
 }
 ```
 
+## C++
+
+```c++
+#include <cstdlib>
+#include <iostream>
+
+int main(int, char**)
+{
+  std::cout << "Hello World!\n" << std::flush;
```

Comment:
In common C++ hello world examples, it seems to be more common to use `std::endl` instead of an explicit "\n" and `std::flush`.

Reply:
My main problem with std::endl is that is platform dependent and do different things in Windows/Mac or GNU/Linux.

https://accu.org/journals/overload/27/149/sharpe_2619/